cfme/tests/storage/test_manager.py

```diff
@@ -40,3 +41,48 @@ def test_manager_navigation(collection_manager):
     assert view.is_displayed
 
     manager.refresh()
+
+
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(OpenStackProvider))
+def test_storage_manager_edit_tag(collection_manager):
+    """ Test add and remove tag to storage manager
+
+    prerequisites:
+        * Storage provider
+
+    Steps:
+        * Add tag and check
+        * Remove tag and check
+    """
+
+    manager = collection_manager[1]
+    # add tag with category Department and tag communication
+    manager.add_tag('Department', 'Communication')
+    tag_available = manager.get_tags()
+    assert tag_available[0].display_name == 'Communication'
+    assert tag_available[0].category.display_name == 'Department'
+
+    # remove assigned tag
+    manager.remove_tag('Department', 'Communication')
+    tag_available = manager.get_tags()
+    assert not tag_available
+
+
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(OpenStackProvider))
```

Comment:
**REQUIRED**
```python
@pytest.mark.provider([OpenStackProvider])
```

Reply:
@quarckster added changes in pytestmark better than each test